src/scripts/OSFramework/OSUI/Pattern/DatePicker/scss/_datepicker.scss

```diff
@@ -34,9 +34,9 @@
 	}
 
 	// Hide the platform input which is set as hidden by the library and we're change it into the expected type, however we do not want it visible since library will add a clone to better deal with the selected dates.
-	& > span > input {
+	& span > input.flatpickr-input {
```

Comment:
Why we don't simplify this even more by changing this to 
".flatpickr-input:first-of-type"
?

Reply:
@JoaoFerreira-FrontEnd I kept the same approach has before to guarantee a direct selector of the element, the first input with class. We never know if a user can add an extra input with this clas, in SS or by JS.

Reply:
What will we gain from adding specificity on the selector and removing the `!important`, when we never want to show this input that is cloned and added by the library? Just to try to have both perspectives since keeping the same approach shouldn't work as a justification. What are the pros and cons?
@bmarcelino-fe @JoaoFerreira-FrontEnd 
cc: @BenOsodrac

Reply:
@gnbm I can't remember the rational of using the !important here. In my perspective, if we have a solution to avoid it, it's better. With this approach and based on my tests, the input cloned is always hidden.
Related to the **:first-of-type** is to make sure that only the cloned element is affected. If the user adds the class .flatpickr-input to the input for some reason, we will hide the element from **dom**.

Reply:
The first lesson of CSS is always: don't use !important, use specificity and cascase if you can.

We should just use !important when there's absolutely no other way to fix something. And here we have other ways.

Its a bad practice, that will increase the specificity of this rule to the maximum, and probably trigger warnings on CSS audit tools.

Reply:
So, I agree with @bmarcelino-fe solution here.

Reply:
I still don't understand why we need the selector to have the span and the element input. But if we put ourselves in a weak position by removing let's go on.